src/ResourceManager/ServiceBus/Commands.ServiceBus/Cmdlets/Queue/NewAzureServiceBusQueue.cs

```diff
@@ -68,6 +68,11 @@ public class NewAzureRmServiceBusQueue : AzureServiceBusCmdletBase
         [ValidateNotNullOrEmpty]
         public bool? DeadLetteringOnMessageExpiration { get; set; }
 
+        [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = "Enable Batched Operations - value that indicates whether server-side batched operations are enabled")]
+        [ValidateSet("TRUE", "FALSE", IgnoreCase = true)]
+        [ValidateNotNullOrEmpty]
```

Comment:
@v-Ajnava please remove the `ValueFromPipelineByPropertyName` property, the `ValidateSet` attribute, and the `ValidateNotNullOrEmpty` attribute

Reply:
Resolved